packages/vsx-registry/src/browser/style/index.css

```diff
@@ -49,6 +49,9 @@
 .theia-vsx-extension {
     display: flex;
     flex-direction: row;
+    padding: 0 0 0 16px;
```

Comment:
We should not use concrete numbers, look how padding and line height are computed for other stlyes in this file.